packages/create-react-native-library/src/input.ts

```diff
@@ -84,6 +85,11 @@ const TYPE_CHOICES: {
   value: ProjectType;
   description: string;
 }[] = [
+  {
+    title: 'Nitro module',
+    value: 'nitro-module',
+    description: 'nitro for faster modules',
```

Comment:
The description should say what it is for. What can I use nitro modules for? For wrapping native APIs? Writing native views? Both?